braindecode/models/tidnet.py

```diff
@@ -0,0 +1,251 @@
+from torch import nn
+from math import ceil
+from torch.nn import init
+from torch.nn.utils import weight_norm
+from .modules import Ensure4d, Expression
+
+import torch
```

Comment:
```suggestion
from math import ceil

import torch
from torch import nn
from torch.nn import init
from torch.nn.utils import weight_norm

from .modules import Ensure4d, Expression
```

import order should be from standard to specific ie std lib -> local relative imports